lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -77,26 +80,19 @@ define((require, exports, module) ->
 
     getCurrentFormula: ->
       try
-        if @curStartId != @curEndId
-          startSequence = EquationRangeHandler.BEGIN_EQUATION_TOKEN_SEQUENCES[@curStartId].slice(0).reverse()
-          startString = (token.value for token in startSequence).join("")
-          endSequence = EquationRangeHandler.END_EQUATION_TOKEN_SEQUENCES[@curEndId]
-          endString = (token.value for token in endSequence).join("")
-          return { title: "Error!", content: "Starting and ending sequences don't match: #{startString} and #{endString}" }
-        { row: startRow, column: startColumn } = @curInnerRange.start
-        tokenIterator = new ConstrainedTokenIterator(@editor.getSession(), @curInnerRange, startRow, startColumn)
+        { row: startRow, column: startColumn } = @currentRange.start
```

Comment:
Done: now displaying incorrect state to a user. If you think I should report it to our server, I'll do it in a separate pull request.